convert/convert_bert.go

```diff
@@ -137,7 +136,7 @@ func replaceFunc[S ~[]E, E any](s S, f func(E) E) S {
 	return s
 }
 
-func (p *bert) Tensors(ts []Tensor) []llm.Tensor {
+func (p *bert) Tensors(ts []Tensor, nameFunc NameFunc) []llm.Tensor {
```

Comment:
Why add a new param here `nameFunc` vs what we had before with `p.tensorName` etc?